public/docs/ts/latest/guide/index.jade

```diff
@@ -80,37 +81,28 @@ table(width="100%")
 
   After reading the above sections, you can skip to any other pages on this site.
 
-:marked
   ## Code samples
 
   Each page includes code snippets that you can reuse in your applications.
   These snippets are excerpts from a sample application that accompanies the page.
 
-block example-links
-  :marked
-    Look for a link to a running version of that sample near the top of each page,
-    such as this <live-example name="architecture"></live-example> from the [Architecture](architecture.html) page.
+  Look for a link to a running version of that sample near the top of each page,
+  such as this <live-example name="architecture"></live-example> from the [Architecture](architecture.html) page.
+  <p if-docs="ts">
+  The link launches a browser-based code editor where you can inspect, modify, save, and download the code.
+  </p>
 
-    The link launches a browser-based code editor where you can inspect, modify, save, and download the code.
-
-:marked
   A few early pages are written as tutorials and are clearly marked as such.
   The rest of the pages highlight key points in code rather than explain each step necessary to build the sample.
   You can always get the full source through the #{_liveLink}.
 
   ## Reference pages
 
-  The [Cheat Sheet](cheatsheet.html) lists Angular syntax for common scenarios.
-
-  The [Glossary](glossary.html) defines terms that Angular developers should know.
-
-  The [API Reference](../api/) is the authority on every public-facing member of the Angular libraries.
-
-  ## Feedback
-
-  We welcome feedback! 
-
-  Use the [angular.io Github repo](https://github.com/angular/angular.io) for **documentation** issues and pull requests.
+  - The [Cheat Sheet](cheatsheet.html) lists Angular syntax for common scenarios.
+  - The [Glossary](glossary.html) defines terms that Angular developers should know.
+  - The [API Reference](../api/) is the authority on every public-facing member of the Angular libraries.
 
+  ## We welcome feedback! 
 
-  Use the [Angular Github repo](https://github.com/angular/angular) to report issues with **Angular** itself.
+  - Use the [site Github repo](!{_ngDocRepoURL}) for **documentation** issues and pull requests.
```

Comment:
Github -> GitHub

(Global. Not introduced in this PR, but while you're there...)